pyproject.toml

```diff
@@ -59,7 +59,7 @@ python-jose = "~3.0"
 pyyaml = ">=4.2b4"
 redis = "~3.2"
 requests = "~2.21"
-shipchain-common = "~1.0"
+shipchain-common = "~1.0.9"
```

Comment:
why is it locked now to a specific minor version?

Reply:
Because the recent changes require shipchain-common >= 1.0.9, but we still want to lock to the _patch_ version